### ISSUE TITLE
[Feature request] Select multiples entries for drag and drop in categories or across databases

### ISSUE THREAD

**Issue by Maxiride** [keeweb/keeweb#729]
I was trying to select multiple entries to organize them into categories but it seems like I can only select + drag and drop one at time . Am I dumb? If not it would be a nice feature I think.

Furthermore it would be cool to drag and drop across databases too!

**Comment by antelle** [comment]
Yes, it should be done with cmd (ctrl) or shift keys. Dragging across databases is a bit more complicated though.

**Comment by Maxiride** [comment]
As a workaround I used KeePassX, but let's face it Keeweb is way more cool :+1:

**Comment by Pfeilgiftfrosch** [comment]
Dear all
I search for the same solution to select and drag&drop multiple entries at the same time.
Unfortunately it doesn't work with cmd, ctrl, shift, or mous selection (Using OS X high sierra).
Is there any solution yet?

**Comment by antelle** [comment]
@Pfeilgiftfrosch it's open, this means it's not done, so there's no way to do it in keeweb now.

**Comment by Pfeilgiftfrosch** [comment]
Thanks for your quick reply. Then i'll try the workaround with macpass or KeePassX
